department-of-veterans-affairs/caseflow
# Queue | Deactivate Case Details link for paper cases

Resolves #5394 

### Description
This disables the case details link and displays a message for paper cases

### Acceptance Criteria 
- [ ] Paper cases have no link to appeal detail view, say "This is a paper case" underneath disabled link

### Testing Plan
1. Re-seed FACOLS (`rake local:vacols:seed`)
2. Go to `/queue` as `BVAAABSHIRE`
3. Confirm `Nola L Stroman (994806951)` link is disabled, paper case note appears
4. Search for `994806951` in case search, confirm Nola Stroman is returned, link is disabled, paper case note appears
